### PR TITLE
PE-2920: fix firefox last modified date

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**

### CODE REVIEW

File: lib/src/utils.dart [hunk 1]
```diff
@@ -32,15 +32,23 @@ Future<List<PlatformFile>> filePathsToPlatformFiles(
...
-      readStream: readStream,
-      size: file.existsSync() ? file.lengthSync() : 0,
-    );
+  Stream<Uint8List> Function([int?, int?])? readStream,
+) async {
+  DateTime? lastModified;
+  try {
+    lastModified = file.lastModifiedSync();
+  } catch (e) {
+    // ignore
```
Comment (matibat) [inline lib/src/utils.dart:45]: We should have test for this case if possible.

File: lib/_internal/file_picker_web.dart [hunk 2]
```diff
@@ -175,19 +190,21 @@ class FilePickerWeb extends FilePicker {
     }
   }
 
-  Stream<List<int>> _openFileReadStream(File file) async* {
+  Stream<Uint8List> _openFileReadStream(File file,
```
Comment (matibat) [inline lib/_internal/file_picker_web.dart:193]: This method seems hard to test. We might need a tech debt to make it testable

**Review by matibat — COMMENTED** [review]
Looking good. I'll approve when I'm done with the other PRs

**Comment by matibat** [comment]
It seems there are conflicts here

File: lib/_internal/file_picker_web.dart (hunk 2 above)
Comment (thiagocarvalhodev) [reply lib/_internal/file_picker_web.dart:193]: You're correct. I'll create one to revisit the tests on file picker.

Comment (thiagocarvalhodev) [reply lib/_internal/file_picker_web.dart:193]: PE-2936

File: lib/src/utils.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/src/utils.dart:45]: PE-2936

**Review by matibat — APPROVED** [review]
LGTM - I couldn't mark the comments as resolved

**Pull request merged**